packages/ceramic-core/src/ceramic.ts

```diff
@@ -117,13 +122,19 @@ class Ceramic implements CeramicApi {
    * @param config - Ceramic configuration
    */
   static async create(ipfs: Ipfs.Ipfs, config: CeramicConfig = {}): Promise<Ceramic> {
-    LoggerProvider.init({
+    const loggerOptions = LoggerProvider.init({
       level: config.logLevel? config.logLevel : 'silent',
       component: config.gateway? 'GATEWAY' : 'NODE',
-      outputToFiles: config.logToFiles,
-      outputPath: config.logPath
     })
 
+    if (config.logToFiles) {
+        LoggerProvider.addPlugin(
+            config.logToFilesPlugin.plugin,
+            loggerOptions,
+            config.logToFilesPlugin.options
+        )
+    }
+
```

Comment:
@simonovic86 I am having the `init` method return the provider's options here because we add the default options and I think it makes sense to have that passed to the plugin when it gets added.

But this paradigm means there is no single source of truth for the state of options between plugins if we were to add more. So I think it would be best if we can make the options immutable from the perspective of plugins.

Reply:
Yeah, I agree. I think that we can use `Object.freeze()` in the `init` method and prevent options from being changed for now. Makes sense?